velox/functions/lib/window/NthValueBase.h

```diff
@@ -111,8 +96,7 @@ class NthValueFunction : public exec::WindowFunction {
       return;
     }
 
-    vector_size_t constantOffsetValue =
-        static_cast<vector_size_t>(constantOffset_.value());
+    vector_size_t constantOffsetValue = constantOffset_.value();
```

Comment:
Can this be auto ? And also should this be const ?

Reply:
Updated, thanks.